_blog-src/_posts/2015-07-07-fuse-apis.markdown

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title:  "Manage Fuse APIs with apiman"
+date:   2015-07-07 15:15:15
+author: Stefan Bunciak
+categories: api management jboss fuse
+---
+
+This article aims to provide a short guide on how to get API Management capabilities provided by apiman to work with JBoss Fuse, a lightweight, flexible, integration platform that is based on [Apache Camel](http://camel.apache.org), an implementation of many of the most commonly used enterprise integration patterns (EIP).
+
+<!--more-->
+
+Creating API provider in JBoss Fuse
+====================================
+
+The following few steps should quickly get you started with deploying applications into JBoss Fuse. For demonstration purposes I'm using a sample REST application (quickstart) shipped along with Fuse installation. For more information about developing applications for JBoss Fuse inspect [developer materials](http://www.jboss.org/products/fuse/developer-materials/#!project=fuse) and other [resources](http://www.jboss.org/products/fuse/resources/) at jboss.org.
+
+### Installation 
+To obtain a distribution of JBoss Fuse please visit the JBoss Fuse [download page](http://www.jboss.org/products/fuse/download/).
+Installation of JBoss Fuse is very simple, as the only thing you need to do is to unzip the distribution, e.g. 
+
+{% highlight sh %}
+unzip jboss-fuse-full-6.2.0.redhat-133.zip
+{% endhighlight %}
+
+You might want to also enable the default admin user by uncommenting the last line in `<fuse_dir>/etc/users.properties`. Without a user configured you won't be able to log into Fuse management console.
+
+### Server startup
+
+To start Fuse simply execute the following command from `<fuse_dir>/bin`:
+
+{% highlight sh %}
+./fuse
+{% endhighlight %}
+
+Once JBoss Fuse is loaded proceed by creating a new Fuse Fabric instance. To do that execute `fabric:create` command from Fuse CLI:
+
+{% highlight sh %}
+JBossFuse:karaf@root> fabric:create
+{% endhighlight %}
+
+After Fuse Fabric is up and running, log into Hawt.io management console, it should be by default available at [http://localhost:8181](http://localhost:8181). 
+In Fabric perspective, under Containers tabs click on the _Create_ button to open the _Create New Container form_. 
+
+![List of Fabric containers](/blog/images/2015-07-07/fabric.png)
+
+On the _Create New Container_ page fill in the name you wish to use for your container (e.g. my-rest-container) and be sure to select 'rest' profile for it. This will ensure that Fabric will pickup the REST Quickstart and deploy it to this container. Finish the operation by hitting the _Create and start container_ button.
```

Comment:
- This will ensure that Fabric will pickup the REST Quickstart and deploy it to this container
- This will ensure that Fabric picks up the REST Quickstart and deploys it to this container